doc/explanation/networks.md

```diff
@@ -1,7 +1,107 @@
- - [ovn](#network-ovn): Creates a logical network using the OVN software defined networking system.
- - [physical](#network-physical): Provides preset configuration to use when connecting OVN networks to a parent interface.
+{ref}`network-bridge`
+: A network bridge creates a connection between different networks or network segments.
+
+  In LXD context, the `bridge` network type creates an L2 bridge that connects the instances that use it and makes it possible to pass traffic between them.
+  The bridge can also provide local DHCP and DNS.
+
+  This is the default network type.
+
+{ref}`network-ovn`
+: {abbr}`OVN (Open Virtual Network)` is a software-defined networking system that supports virtual network abstraction.
+  You can use it to build your own private cloud.
+  See https://www.ovn.org/ for more information.
+
+  In LXD context, the `ovn` network type creates a logical network.
+  To set it up, you must install and configure the OVN tools.
+  In addition, you must create an uplink network that provides the network connection for OVN.
+  As the uplink network, you should use one of the external network types.
+
+### External networks
+
+External networks use network interfaces that already exist.
+Therefore, LXD has limited possibility to control them.
+
+The main purpose for using external networks is to provide an uplink network through a parent interface.
+
+LXD supports the following network types:
+
+{ref}`network-macvlan`
+: Macvlan is a virtual {abbr}`LAN (Local Area Network)` that can assign several IP addresses to the same network interface, basically splitting up the network interface into several sub-interfaces with their own IP addresses.
+  The IP addresses are assigned based on randomly generated MAC addresses.
+
+  In LXD context, the `macvlan` network type provides a preset configuration to use when connecting instances to a parent Macvlan interface.
+
+{ref}`network-sriov`
+: {abbr}`SR-IOV (Single root I/O virtualization)` is a hardware standard that allows a single network interface to appear as several virtual network interfaces in a virtualized environment.
+
+  In LXD context, the `sriov` network type provides a preset configuration to use when connecting instances to a parent SR-IOV interface.
+
+{ref}`network-physical`
+: The `physical` network type connects to an existing physical network, which can be a network card or a bridge, and serves as an uplink network for OVN.
+
+  It provides a preset configuration to use when connecting OVN networks to a parent interface.
+
+## Recommendations
+
+Which network type to choose depends on your specific use case.
+In general, if you can use a fully controlled network, you should do so because it provides more functionality than using a network device.
+In addition, networks are easy to configure, and you can reuse the same network for several instances without repeating the configuration.
```

Comment:
the "In addition" here makes it seem like we are still talking about the benefits of the fully controlled network types, but are in actuality talking about the benefit/purpose of using networks themselves.

Howabout re-organising this section slightly to start with the general points first and then move onto the type specific points. So:

> You can reuse the same network for multiple instances avoiding the need to repeat the configuration in each instance NIC.
> Which network type to choose depends on your specific use case.
> In general, if you can use a fully controlled network, you should do so because it provides more functionality than using a network device.

Reply:
I'm not sure. Perhaps we need to clarify the terminology. It is my understanding that any network that exists in the database is a "managed" network (so that covers all the network types in the documentation). Then I understood that you had then further classified managed networks into two categories; "fully controlled" and the implicit "not-fully controlled" (as these terms are new).

Then the unmanaged networks are those that appear in the `lxc network ls` output with the "MANAGED" column value of "no", e.g.

```
lxc network ls
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
|      NAME       |   TYPE   | MANAGED |      IPV4      |          IPV6           | DESCRIPTION | USED BY |  STATE  |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
| br-int          | bridge   | NO      |                |                         |             | 0       |         |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
| enp0s31f6       | physical | NO      |                |                         |             | 0       |         |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
| enx00249b73f3ba | physical | NO      |                |                         |             | 0       |         |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
| lxdbr0          | bridge   | YES     | 10.64.199.1/24 | fd42:bafd:ac21:9f::1/64 |             | 2       | CREATED |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
| wlp0s20f3       | physical | NO      |                |                         |             | 0       |         |
+-----------------+----------+---------+----------------+-------------------------+-------------+---------+---------+
```

But then if you change:

> In general, if you can use a fully controlled network, you should do so because it provides more functionality than using a network device.

To be:

> In general, if you can use a managed network, you should do so because it provides more functionality than using a network device.

Then this is isn't true, because using a managed macvlan network, for example, doesn't provide any additional functionality over just using the macvlan NIC type with a parent setting. 

The extra functionality only comes when using a fully-controlled managed network (as those provide extra functionality such as DHCP, DNS etc).